packages/mural-client/src/index.ts

```diff
@@ -167,7 +167,39 @@ const optionsParams = (
   return params;
 };
 
-// ====
+const buildCrossWorkspaceParams = (q:string, createdSince?: number | undefined, createdUntil?: number | undefined, createdFilterStrict?: boolean | undefined, updatedSince?: number | undefined, updatedUntil?: number | undefined, updatedFilterStrict?: boolean | undefined) => {
```

Comment:
By using question mark at the end of the parameter name, `| undefined` is not needed